pkg/mac/scripts/horizon-cli-uninstall.sh

```diff
@@ -31,3 +31,14 @@ rm -f $DESTDIR/share/man/*/man1/hzn.1
 echo "Removing $SRCDIR/horizon-cli ..."
 rm -rf $SRCDIR/horizon-cli
 
+# remove hzn_bash_autocomplete.sh that agent-install.sh put in .bashrc and .bash_profile
+if [ -f "${HOME}/.bash_profile" ]; then
+        if [[ "A`grep 'hzn_bash_autocomplete' ~/.bash_profile`" != "A"  ]];then
+                sed -i "" "/hzn_bash_autocomplete/d" ~/.bash_profile
```

Comment:
The full script name is `hzn_bash_autocomplete.sh`, please use that. Same comment applies to .bashrc below.